Exercises/numbers/numbers.js

```diff
@@ -1,3 +1,65 @@
-function extractNumbers (text) {
-  return null;
-}
\ No newline at end of file
+// # Numbers
+// 1. Assign by hand same value to 3 variables in binary, octal and hexadecimal format.
+let binary = '1111';
+let octal = '17';
+let hexadecimal = '0xf';
+
+binary = parseInt(binary, 2);
+octal = parseInt(octal, 8);
+hexadecimal = parseInt(hexadecimal, 16);
+
+console.log(binary == octal, octal == hexadecimal, hexadecimal == binary);
+
+// 2. Implement function `extractNumbers` which will leave only numbers in passed string.
+function extractNumbers(str) {
+  let resultString = '';
+  for (let char in str) {
+    if (isNaN(parseInt(str))){
+      str = str.slice(1);
+    }
+    else {
+      resultString += str[0];
+      str = str.slice(1);
+    }
+  }
+  return resultString;
+}
+
+console.log(extractNumbers('123abc456fgh'));
+
+// 3. Declare a variable with decimal value and convert to binary, octal and hexadecimal.
+let decimal = 15;
+
+console.log(`binary value is '${decimal.toString(2)}';
+octal value is '${decimal.toString(8)}';
+hexadecimal value is '${decimal.toString(16)}'`);
+
+
+// 4. use `Math.random` to generate random value from `0` to `100`.
+function randomBetweenZeroAndHundred() {
+  return Math.floor(Math.random() * 100 + 1);
+} 
+
+console.log('Random number: ' + randomBetweenZeroAndHundred());
+
+// # Additional
+// 1. Make function which will take a array with numbers as a parameter and
+//    calculate average number and round to hundredths.
+function average(arr) {
+  let sum = 0;
+  for (let i = 0; i < arr.length; i++){
```

Comment:
perfect. There is good method for it `reduce`. If you want you can try to rewrite